src/components/AssetManagementPanel/AssetManagementForm/FormVariants/EndorseBeneficiary/EndorseBeneficiary.test.tsx

```diff
@@ -14,17 +14,17 @@ describe("Endorse Owner", () => {
           formAction={AssetManagementActions.EndorseBeneficiary}
           setFormActionNone={() => {}}
           tokenRegistryAddress="0xdA8DBd2Aaffc995F11314c0040716E791de5aEd2"
-          beneficiary="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
+          nominee="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
           holder="0xa61B056dA0084a5f391EC137583073096880C2e3"
-          handleTransfer={() => {}}
+          handleBeneficiaryTransfer={() => {}}
           beneficiaryEndorseState={FormState.UNINITIALIZED}
         />
       );
 
-      const beneficiaryComponent = container.getByTestId("editable-input-owner");
-      const holderComponent = container.getByTestId("editable-input-holder");
+      const nomineeComponent = container.getByTestId("non-editable-input-nominee");
+      const holderComponent = container.getByTestId("non-editable-input-holder");
```

Comment:
do we have a case like this anymore?
the test description doesnt match up with what u are testing